libr/util/udiff.c

```diff
@@ -273,6 +273,12 @@ R_API char *r_diff_buffers_unified(RDiff *d, const ut8 *a, int la, const ut8 *b,
 	}
 	if (!fa || !fb) {
 		R_LOG_ERROR ("fafb nul");
+		if (fa) {
```

Comment:
No need to if(x)free(x)

free accepts null

Reply:
it's still early, forgot that 😂 will change